### PR TITLE
EN 1418 fix groups detail

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/users/common/UserForm.js [hunk 1]
```diff
@@ -19,11 +19,14 @@ import FormLabel from 'ui/common/form/FormLabel';
 const EDIT_MODE = 'edit';
 const NEW_MODE = 'new';
 
+const minLength8 = minLength(8);
+const maxLength20 = maxLength(20);
+
 export const renderStaticField = (field) => {
   const { input, label, name } = field;
-  const fieldValue = (input.value.title) ? input.value.title : input.value;
+  let fieldValue = (input.value.title) ? input.value.title : input.value;
```
Comment (nicpuddu) [inline src/ui/users/common/UserForm.js]: use || instead of a ternary
```
let fieldValue = input.value.title || input.value;
```

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**